src/psyclone/gocean1p0.py

```diff
@@ -493,7 +498,7 @@ def __init__(self, parent=None,
                 "Invalid loop type of '{0}'. Expected one of {1}".
                 format(self._loop_type, const.VALID_LOOP_TYPES))
 
```

Comment:
Worth a comment to say why we need the symbol table of the InvokeSchedule?

Reply:
A yes, this is a subtle but important change, I will add a comment.